kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -513,6 +513,12 @@ def _init_and_run_proof(_init_problem: tuple[str, str]) -> tuple[bool, list[str]
         proof_id = f'{_init_problem[0]}.{_init_problem[1]}'
         llvm_definition_dir = foundry.out / 'kompiled-llvm' if use_booster else None
 
+        def _check_vacuous(cterm: CTerm) -> bool:
```

Comment:
This can be a `staticmethod` of the class.

Reply:
Right, this is not in class scope. Then just consider moving it to the top level (there's no reason for it to be nested two levels.).